pkgs/development/tools/pnpm/fetch-deps/pnpm-config-hook.sh

```diff
@@ -24,7 +24,12 @@ pnpmConfigHook() {
 
     echo "Installing dependencies"
 
-    pnpm install --offline --frozen-lockfile --ignore-script
+    if [[ -n "$pnpmWorkspace" ]]; then
+        extraPnpmInstallFlags="--filter=$pnpmWorkspace"
+    fi
```

Comment:
If we set this already, why are we doing the buildPhases above by hand? I am confused

Reply:
> If we set this already, why are we doing the buildPhases above by hand? I am confused

You are confused because (surprisingly) we don't have a `pnpm.buildHook`. If the project you try to package doesn't have a specific workspace you are interested in, using `npmHooks.npmBuildHook` probably will do the job. With a specific workspace you pretty much have to use `pnpm --filter=.. build`. Perhaps we should add `pnpm.buildHook`.